### ISSUE TITLE
generic HTTP consumer: customization of json keys

### ISSUE THREAD

**Issue by amolari** [F5Networks/f5-telemetry-streaming#114]
### Is your feature request related to a problem? Please describe.
Some eventlog receivers expect to receive the message/event and event category as value of specific keys. For example: Datadog (keys= message, service  as per doc https://docs.datadoghq.com/api/latest/logs/#send-logs).

### Describe the solution you'd like
When sending event logs with the generic HTTP consumer, the event message is the value of key `data`.
The event category is the value of key `telemetryEventCategory`. We need to be able to customize those 2 keys values'.
That way we'd be able to sent the event message as value of key (for example) `message`. The same for the event category, that we could send as value of key (for example) `service`.

### Describe alternatives you've considered
Send messages through a system such as logstash, that allows to rewrite the json payload. That works but is not optimal, as it requires an intermediate system.

**Comment by dstokesf5** [comment]
I agree that this would be very useful, and I was surprised that this was not already in our product backlog. I have added the ability to customize the structure of the Generic_HTTP consumer payload to our internal product backlog as AUTOTOOL-2312.

**Comment by dstokesf5** [comment]
This issue was resolved as part of the 1.20.0 release.

**Comment by amolari** [comment]
@dstokesf5 
> This issue was resolved as part of the 1.20.0 release.

are you sure? I do not see any code change for this consumer in 1.20.0

**Comment by dstokesf5** [comment]
An example of customizing the payload of the Generic_HTTP consumer can be found here: https://clouddocs.f5.com/products/extensions/f5-telemetry-streaming/latest/declarations.html#customizing-the-generic-http-consumer-payload
